jira-config/jira-config-ui/src/main/resources/JIRA/JIRAConfigSheet.xml

```diff
@@ -78,4 +90,182 @@
+        <number>4</number>
+        <prettyName>Parse content</prettyName>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      </parse>
+      <use>
+        <cache>0</cache>
+        <disabled>0</disabled>
+        <displayType>select</displayType>
+        <freeText>forbidden</freeText>
+        <largeStorage>0</largeStorage>
+        <multiSelect>0</multiSelect>
+        <name>use</name>
+        <number>3</number>
+        <prettyName>Use this extension</prettyName>
+        <relationalStorage>0</relationalStorage>
+        <separator> </separator>
+        <separators>|, </separators>
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>currentPage|onDemand|always</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </use>
+    </class>
+    <property>
+      <cache>long</cache>
+    </property>
+    <property>
+      <code>require(['jquery'], function($) {
+    const urlMapping = JSON.parse($('#jiraConfig')[0].dataset.authDisplayerUrls)
+    /**
+     * Load or reload the displayer for the authenticator configuration
+     * @param {*} element the parent container for the displayer content
+     */
+    async function updateDisplayer(element) {
+        const id = $(element).parent().find(".jira-server-id &gt; input")[0].value
+        const authType = $(element).parent().find("select.jira-config-authentication-type")[0].value
+        const url = urlMapping[authType]
+        let body = ''
+        if (url !== undefined) {
+            try {
+                const response = await fetch(`${url}?id=${id}&amp;authType=${authType}`);
+                if (!response.ok) {
+                    console.error("can't get displayer content")
+                    return
+                }
+                body = await response.text()
+            } catch (e) {
+                console.error(e);
+                return
+            }
+        }
+        element.innerHTML = body
+    }
+
+    // First load of the authenticator displayer
+    for (const element of $('.jira-auth-configuration')) {
+        new Promise(() =&gt; updateDisplayer(element))
+    }
+
+    // Reload the authenticator displayer if we change the authenticator
+    for (const element of $('.jira-config-authentication-type')) {
+        $(element).on('change', async (event) =&gt; {
+            const e = $(element).parent().parent().find('td.jira-auth-configuration')[0]
+            await updateDisplayer(e)
+        })
+    }
+
+    // Avoid that the on the reload of the page we cancel the async save for the authentication configuration
+    // The save of the main configuration will be done after we saved all authenticator configuration
+    $('#jiraConfig input.button').on('click', (e) =&gt; {
+        e.preventDefault()
+    });
+
+    // Save the authenticator configuration
+    $('#jiraConfig input.button').on('click', async (e) =&gt; {
```

Comment:
Feels odd to have two selectors to `#jiraConfig input.button`, once to call preventDefault, then another time for the business logic. WDYT of merging them?